tiering/ctl.h

```diff
@@ -2,10 +2,21 @@
 /* Copyright (C) 2021 Intel Corporation. */
 
 #pragma once
+#include <memkind_memtier.h>
+
+#include <stddef.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+typedef struct tier_draft {
+    char *kind_name;
+    char *pmem_path;
+    size_t pmem_size;
+    unsigned ratio_value;
+} tier_draft;
```

Comment:
tier_draft -> ctl_tier_cfg ? not sure something with control and tier configuration ?